content/pt-br/docs/reference/glossary/cadvisor.md

```diff
@@ -0,0 +1,18 @@
+---
+title: cAdvisor
+id: cadvisor
+date: 2021-12-09
+full_link: https://github.com/google/cadvisor/
+short_description: >
+  Ferramenta que provê o entendimento do uso de recursos e características de desempenho para contêineres
```

Comment:
I like "fornece uma compreensão" better than "provê o entendimento", like you wrote here:
https://github.com/kubernetes/website/blob/86d8e75224ee876c7989fbd5b7375fa5ae6fb157/content/pt-br/docs/reference/glossary/cadvisor.md#L12